ui/client/documents/index.js

```diff
@@ -639,6 +385,10 @@ const ViewDocumentsGrid = withStyles(() => ({
                 onPageChange={handlePageChange}
                 paginationMode="server"
                 rowCount={totalRowsCount}
+                disableColumnFilter
+                sortingMode="server"
+                onSortModelChange={handleSortChange}
+                disableColumnMenu
```

Comment:
Nit/nice to have (brainstorming out loud):

I wonder if we should add the column menu, if this is what I think this is. That is, it would be nice to users to select which columns to display if they can, and allow to sort by any of those. Or maybe this isn't what I think it is.

Also, this is very optional for our purposes, but in general if we were doing any other UI and this was easy to do it would be very nice (especially as the API would allow sorting by arbitrary fields- see my previous comments).

Reply:
Removing the column menu was actually the very last thing I did, for a few reasons:
1. There are only four columns at the moment, so I didn't think we need to allow the removing columns thing. If Brandon wants to include more columns in the future, this might be worth reconsidering.
2. Filtering isn't enabled on the backend, so we don't have filtering actually working (another default feature of the column menu).
3. Most significantly, if you have the sort buttons (ASC/DESC in the menu) enabled at all, you have to include the "unsort" button. This doesn't do anything at the moment and I don't really even know what unsorted would mean. Since we already have the arrow, including the menu would basically just be giving us redundant ASC/DESC buttons and a nonfunctional unsort button.